docs/v0.9/en/user-guide/operations/compaction.md

```diff
@@ -0,0 +1,142 @@
+
+### SST files
+
+SSTs are sorted files generated when memtables are flushed to persistent storage, such as disks and object stores.
+
+In GreptimeDB, data rows in SST files are organized by [tags](/user-guide/concepts/data-model.md) and timestamps, as illustrated below. Each SST file covers a specific time range. When a query specifies a time range, GreptimeDB retrieves only the relevant SST files that may contain data within that range, rather than loading all persisted files.
+
+![SST layout](/compaction-sst-file-layout.jpg)
+
+Typically, the time ranges of SST files do not overlap during real-time writing workloads. However, due to factors like row deletions and out-of-order writes, SST files may have overlapping time ranges, which can affect query performance.
+
+### Time window
+
+Time-series workloads present prominent "windowed" pattern in that most recently inserted rows are more possible to be read. Thus GreptimeDB logically partitioned the time axis into different time windows and we are more interested in compacting those SST files fall into the same time window. 
+
+Time window for some specific table is usually inferred from the most-recently flushed SST files or users may manually specify the time window if TWCS is chosen.
+
+GreptimeDB has a preset collection of window sizes that are:
+- 1 hour
+- 2 hours
+- 12 hours
+- 1 day
+- 1 week
+- 1 year
+- 10 years
+
+If time window is not specified, GreptimeDB will infer it when the first compaction happens by selecting the minimum time window size for the above collection that can cover the whole time span of all files to be compacted.
+
+For example, during the first compaction, the time span for all SST files is 4 hours, then GreptimeDB will choose 12 hours as the time window for that table and persist it for later compactions.
+
+GreptimeDB considers the window contains most recently inserted timestamps as **active window** and those previous windows as **inactive windows**.
+
+### Sorted runs
+Sorted runs is a collection of SST files that have sorted and non-overlapping time ranges. 
+
+For example, a table contains 5 SSTs with following time ranges (all inclusive): `[0, 10]`, `[12, 23]`, `[22, 25]`,`[24, 30]`,`[26,33]` and we can find 2 sorted runs:
+
+![num-of-sorted-runs](/compaction-num-sorted-runs.jpg)
+
+
+The number of sorted runs indicates the orderliness of SST files. More sorted runs typically lead to worse query performance. The main goal of compactions is to reduce the number of sorted runs.
+
+### Levels
+
+Databases based on LSM trees may also have levels, keys are merged level by level. GreptimeDB only has 2 levels which are 0 (uncompacted) and 1 (compacted).
+
+## Compaction strategies
+GreptimeDB provides two compaction strategies as mentioned above, but only Time Windowed Compaction Strategy (TWCS) can be chosen when creating tables. Strict window is only available when executing manual compactions.
+
+## Time Windowed Compaction Strategy
```

Comment:
```suggestion
## Time Windowed Compaction Strategy (TWCS)
```

Reply:
fixed in [5328a81f](https://github.com/GreptimeTeam/docs/commit/5328a81f0c42cdc40aecbf168efa50e455f3db0d#diff-2643784f62d50a7674027cfc0acf77e84d673b2407cff812d3f0a1d92962571cR60)